TKeazirian.HTTPServer/Program.cs

```diff
@@ -1,11 +1,63 @@
-﻿namespace TKeazirian.HTTPServer;
+﻿using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace TKeazirian.HTTPServer;
+
 public static class Program
 {
-    private static void Main(string[] args)
+    public static string? IncomingData;
+
+    private static void StartListening()
     {
-        var g = new Greeting();
-        var hour = DateTime.Now.Hour;
-        var greeting = Greeting.GetGreeting(hour);
-        Console.WriteLine(greeting);
+        byte[] bytes = new byte[1024];
+
+        IPHostEntry ipHostInfo = Dns.GetHostEntry(Dns.GetHostName());
+        IPAddress ipAddress = ipHostInfo.AddressList[0];
+        IPEndPoint localEndPoint = new IPEndPoint(ipAddress, 11000);
```

Comment:
Is `11000` the port? Why `11000`? Could the port be something the user gets to decide?

Reply:
Hmm...I assume it is possible but I'm not sure how (yet) it could be done so the user can decide...

I chose port 11000 sort of randomly, from examples I've seen.  I also tried it `1234`.